docs/codeql/codeql-language-guides/abstract-syntax-tree-classes-for-working-with-java-programs.rst

```diff
@@ -3,7 +3,7 @@
 Abstract syntax tree classes for working with Java programs
```

Comment:
As far as I can tell this PR looks good: it's very difficult to see where other references may need to be changed because there are simply so many references to the strings in question, but I went through the `codeql` repo and couldn't see anywhere obvious that you hadn't already changed 👍 

This was the only thing I wasn't sure about which might be missing (since it's referenced with Kotlin and the equivalent title for JavaScript also mentions TypeScript in the title itself):

```suggestion
Abstract syntax tree classes for working with Java and Kotlin programs
```

?

Reply:
Thanks so much for the review @subatoi! Definitely hard to find everything but it does seem we got the main ones